### PR TITLE
[WIP] Add Lif on server and app

### LINKED ISSUE #148: Send tx data for payment instructions from server
Right now the the txData to execute the payment is calculated in the client instead we will send an array of txs to execute for the payment alongside the signedOffer.

The txs in the array should have the fields: to, data, gas and value.

For ETH the client will receive one tx and for Lif two.

Depends on #146

### PR DESCRIPTION
**Pull request by AugustoL**
fix #148

### CODE REVIEW

**Comment by AugustoL** [comment]
@PabloAlice you can push the commits for lif integration here. Now the response from booking request returns txs[] there is no need to calculate the data in teh browser.

File: server/src/models/Booking.js [hunk 1]
```diff
@@ -150,17 +150,18 @@ Booking.method({
     const randomCode = Math.floor((1 + Math.random()) * 10000);
     this.bookingHash = web3.utils.sha3(`${randomCode}${Date.now()}`);
   },
-  generatePaymentAmount: function (ethPrice) {
-    if (typeof ethPrice !== 'number') {
-      throw handleApplicationError('invalidEthPrice');
+  generatePaymentAmount: function (cryptoPrice) {
+    if (typeof cryptoPrice !== 'number') {
+      console.log(typeof cryptoPrice)
```
Comment (beogip) [inline server/src/models/Booking.js]: remove `console.log`

File: server/src/routes/index.js [hunk 2]
```diff
@@ -6,14 +6,20 @@ const {
...
+const { getInstructionsTxs } = require('../services/web3');
 
 const router = express.Router();
 const bookingUrl = '/booking';
 
 router.post(`${bookingUrl}`, async (req, res, next) => {
   try {
     const { signatureData, booking, offerSignature } = await createBooking(req.body);
+    const nights = [];
+    for (let i = booking.from; i <= booking.to; i++)
```
Comment (beogip) [inline server/src/routes/index.js]: could you add curly brackets to make it consistent with the current code style?

**Comment by AugustoL** [comment]
@beogip done.

**Pull request merged**